src/util/network.js

```diff
@@ -6,7 +6,20 @@ const IGNORE_INTERFACES = ['lo0', 'awdl0', 'bridge0'];
 const LOCAL_IPS = ['127.0.0.1', '::1'];
 
 export function isOffline(): boolean {
-  const interfaces = os.networkInterfaces();
+  let interfaces;
+
+  try {
+    interfaces = os.networkInterfaces();
+  } catch (e) {
+    // As of October 2016, Windows Subsystem for Linux (WSL) does not support
+    // the os.networkInterfaces() call and throws instead. For this platform,
+    // assume we are online.
+    if (e.syscall === 'uv_interface_addresses') {
+      return false;
+    } else {
+      throw e;
+    }
+  }
```

Comment:
I like doing that in my code because it makes the flow of execution clear.

Happy to remove the `else` if you'd prefer.